src/nodejs/unit-http/unit.cpp

```diff
@@ -602,7 +603,12 @@ Unit::create_headers(nxt_unit_request_info_t *req, napi_value request)
 
     set_named_property(request, "headers", headers);
     set_named_property(request, "rawHeaders", raw_headers);
-    set_named_property(request, "httpVersion", r->version, r->version_length);
+
+    // trim the "HTTP/" protocol prefix
+    p = (char *) nxt_unit_sptr_get(&r->version);
```

Comment:
Hi @andrey-zelenkov 

I take it this cast is required due to C++? (In C it wouldn't be needed).

Reply:
Yeah, I was also surprised that cast is required here. I was getting following error without it (line number is slightly shifted):

```
npm ERR! ../unit.cpp:611:26: error: invalid conversion from ‘void*’ to ‘char*’ [-fpermissive]
npm ERR!   611 |     p = nxt_unit_sptr_get(&r->version);
npm ERR!       |         ~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~
npm ERR!       |                          |
npm ERR!       |                          void*
```

Reply:
OK.